server/src/main/java/org/candlepin/model/Content.java

```diff
@@ -122,9 +137,18 @@
     @Size(max = 255)
     private String arches;
 
+    @XmlTransient
+    @Column(name = "entity_version")
+    private Integer entityVersion;
+
+    @XmlTransient
+    @Column
+    private Integer locked;
```

Comment:
I didn't even get to the Hibernate level. The issues with juggling were happening at the jdbc level in the migration task -- I'd have to know which DB I was talking to and manually cast/set it appropriately.